src/Workspaces/Core/Portable/Formatting/Engine/TokenStream.cs

```diff
@@ -501,9 +515,39 @@ private bool IsFirstTokenOnLine(TokenData tokenData1, TokenData tokenData2)
             return this.GetTriviaData(tokenData1, tokenData2).SecondTokenIsFirstTokenOnLine;
         }
 
+        /// <summary>
+        /// Gets approximated index bounds for a token with specified position, which will definitely contain
+        /// the token, if it is present in the stream.
+        /// </summary>
+        private void GetTokenIndexBounds(int position, out int lowerBound, out int upperBound)
+        {
+            float avgLength = this.AverageTokenLength;
```

Comment:
1. var preferred.
2. we don't really need the local.  can just cheaply reference the field.  it will all be inlined well.
3. consider not even having a property and just a field (it's not exposed outside this type right)?
4. in general we don't like to abbreviate.  so 'average' instead of 'avg'.

Reply:
inlined